abnf/odata-abnf-testcases.yaml

```
@@ -1811,16 +1811,20 @@ TestCases:
     Rule: boolCommonExpr
     Input: endswith(["Fred","George","Ron"],["George","Ron"])
 
+  - Name: 5.1.1.5.3 indexof
+    Rule: commonExpr
+    Input: indexof(CompanyName,'lfreds')
+
   - Name: 5.1.1.5.4 indexof
-    Rule: boolCommonExpr
+    Rule: commonExpr
     Input: indexof(["Fred","George","Ron"],["George","Ron"]) eq 1
 
   - Name: 5.1.1.5.5 length
-    Rule: boolCommonExpr
+    Rule: commonExpr
```

```suggestion
    Rule: boolCommonExpr
```
The whole expression returns a boolean.